autoware_launch/config/planning/preset/default_preset.yaml

```diff
@@ -0,0 +1,87 @@
+launch:
+  # - arg:
+  #     name: behavior_path_planner_launch_modules
+  #     default: "[behavior_path_planner::Avoidance,
+  #       behavior_path_planner::AvoidanceByLaneChange,
+  #       behavior_path_planner::DynamicAvoidance,
+  #       behavior_path_planner::LaneChange,
+  #       behavior_path_planner::ExternalLaneChange,
+  #       behavior_path_planner::StartPlanner,
+  #       behavior_path_planner::GoalPlanner,
+  #       behavior_path_planner::SideShift]"
+  #     # option: behavior_path_planner::Avoidance
+  #     #         behavior_path_planner::AvoidanceByLaneChange
+  #     #         behavior_path_planner::DynamicAvoidance
+  #     #         behavior_path_planner::LaneChange
+  #     #         behavior_path_planner::ExternalLaneChange
+  #     #         behavior_path_planner::StartPlanner
+  #     #         behavior_path_planner::GoalPlanner
+  #     #         behavior_path_planner::SideShift"
+
+  - arg:
+      name: behavior_velocity_planner_launch_modules
+      default: "[behavior_velocity_planner::CrosswalkModulePlugin,
+        behavior_velocity_planner::WalkwayModulePlugin,
+        behavior_velocity_planner::TrafficLightModulePlugin,
```

Comment:
In order to prevent duplicate descriptions and typos in the behavior_velocity_planner plugin, wouldn't it be better to write it in "commented out format" like before?

Reply:
@tkimura4 Thanks for your comment. I think so. But it seems that it can't do comment out like this since this parameter is definded as a string.

```yaml
  - arg:
      name: behavior_velocity_planner_launch_modules
      default: "[behavior_velocity_planner::CrosswalkModulePlugin,
        behavior_velocity_planner::WalkwayModulePlugin,
        behavior_velocity_planner::TrafficLightModulePlugin,
        behavior_velocity_planner::IntersectionModulePlugin,
      # behavior_velocity_planner::MergeFromPrivateModulePlugin,
        behavior_velocity_planner::BlindSpotModulePlugin,
        behavior_velocity_planner::DetectionAreaModulePlugin,
        behavior_velocity_planner::NoStoppingAreaModulePlugin,
        behavior_velocity_planner::StopLineModulePlugin,
        behavior_velocity_planner::OutOfLaneModulePlugin]"
```

Or do you have any workarounds...?

Reply:
oh... :cry:  Sorry, I don't know the solution either.

In the current string format, is it easy to understand errors caused by duplicate plugin descriptions or typos?

Reply:
> oh... 😢 Sorry, I don't know the solution either.

:pray: 

> In the current string format, is it easy to understand errors caused by duplicate plugin descriptions or typos?

Umm...maybe... It may ouputs error message from node, I guess. :thinking: 

```c++
    RCLCPP_ERROR_STREAM(node.get_logger(), "The scene plugin '" << name << "' is not available.");
```

https://github.com/autowarefoundation/autoware.universe/blob/0d929d408eb3459962c9cd228421a826669705dc/planning/behavior_velocity_planner/src/planner_manager.cpp#L76

Reply:
I think it has better use `std::invalid_argument` instead of `RCLCPP_ERROR_STREAM` in order to notice invalid launch config.

Reply:
@tkimura4 I update xml so that it can align launch flag style in https://github.com/autowarefoundation/autoware_launch/pull/680/commits/096acd81328763243f5acbb19636af4d09b673d2.